cluster/src/dunit/scala/io/snappydata/cluster/ClusterManagerTestBase.scala

```diff
@@ -75,9 +76,32 @@ class ClusterManagerTestBase(s: String) extends DistributedTestBase(s) {
   // this can be used only by jobs running on Lead node
   def sc: SparkContext = SnappyContext.globalSparkContext
 
+  val startNode = new SerializableRunnable() {
+    override def run(): Unit = {
+      val node = ServiceManager.currentFabricServiceInstance
+      if (node == null || node.status != FabricService.State.RUNNING) {
```

Comment:
I could not find also  there is no way to re start the server in the ClusterManagerTestBase. I re-factored the code so that startNode is accessible from all the child classes now and they can use it to re start the server.
later we will need a generic way to stop and start the network servers as well.